packages/uploadthing/src/internal/handler.ts

```diff
@@ -183,13 +183,6 @@ export const buildRequestHandler = <
         cause: `Expected uploadthingHook to be of type 'string', got '${typeof uploadthingHook}'`,
       });
     }
-    if (typeof req.body !== "string") {
-      return new UploadThingError({
-        code: "BAD_REQUEST",
-        message: "Request body must be a JSON string",
-        cause: `Expected request body to be of type 'string', got '${typeof req.body}'`,
-      });
-    }
```

Comment:
we never use the `req.body` - not sure why this check was there in the first place @markflorkowski ?

Reply:
We do use the body, just via `.json()` -- this check is there to make sure it exists I guess. Probably should attempt parsing to make sure it is valid JSON before throwing though. 

![image](https://github.com/pingdotgg/uploadthing/assets/11494384/0a0128bc-95a2-4aec-85f5-c3af2876c240)

Reply:
Aight Yea cause this threw in cases it shouldnt have so i nuked it